### PR TITLE
Update companion apps

### PR DESCRIPTION
**Pull request by darkdragon-001**

### CODE REVIEW

**Comment by darkdragon-001** [comment]
I came here via "Edit on GitHub" link in the [documentation](https://docs.infinitime.io/en/latest/user-documentation/index.html#companion-apps) (using `documentation-reorganization` branch as target), but I am not sure if one should target `main` instead.

**Comment by FintasticMan** [comment]
The `documentation-reorganization` branch hasn't been updated in quite a while, and I'm quite sure a lot of it needs to be redone. Watchmate is already mentioned in the README on `main`, so I'd hold off on this PR until we can get someone who's able to spend the time updating the documentation.

**Comment by darkdragon-001** [comment]
Could you at least point the documentation to `main` for `latest` in the meantime: https://docs.infinitime.io/en/latest/

File: docs/user-documentation/index.md
Comment (FintasticMan) [inline docs/user-documentation/index.md:101]: I think it would be good to replace this with something similar to the README, because the line between desktop and mobile isn't very clear-cut when it comes to Linux.

**Review by FintasticMan — CHANGES_REQUESTED** [review]
I've just had a look, and although not much has happened to the documentation, there was recently a PR merged into `documentation-reorganization` that contained a fix for a broken link. Because of that, I think this PR should also be merged, so that at least 1 issue with the ReadTheDocs is fixed :)

File: docs/user-documentation/index.md
Comment (darkdragon-001) [reply docs/user-documentation/index.md]: I copied the lines from the README

**Review by FintasticMan — APPROVED** [review]

**Comment by darkdragon-001** [comment]
Any updates?

**Comment by FintasticMan** [comment]
It's good to be merged from my side, we just need 1 more InfiniTime member to review it before I can merge it.

**Comment by darkdragon-001** [comment]
How can I see who is an InfitiTime member eligible to review?

@JF002 can this be merged?

**Comment by FintasticMan** [comment]
I don't think GitHub allows you to check the members of the organization. JF's said he can't work on InfiniTime until at least the 1st of August. Perhaps @Avamander or @NeroBurner can review this quickly?

**Review by Avamander — APPROVED** [review]
LGTM

**Pull request merged**
